src/cronjobs/userProfileJobs.js

```diff
@@ -1,26 +1,26 @@
-const { CronJob } = require('cron');
-const moment = require('moment-timezone');
+const { CronJob } = require("cron");
```

Comment:
Please revert the double quotes back to single quotes. I'm assuming it's the linter that's doing this.